torch/csrc/autograd/variable.cpp

```diff
@@ -13,6 +13,7 @@ namespace torch { namespace autograd {
 
 Variable make_variable(at::Tensor data, std::shared_ptr<Function> grad_fn) {
   TORCH_ASSERT(grad_fn);
+  if (!data.defined()) return Variable();
```

Comment:
Are these necessary for the convolution change?

Reply:
Seems this is only needed if you return undefined from a function, which happened briefly when I was experimenting with having buffers be return values. The current patch doesn't need it, so I'll comment this out.